bundles/security/index.rst

```diff
@@ -137,13 +137,13 @@ admin firewall. This is configured in the ``config/packages/security.yaml``:
                 # ...
                 logout:
                     path: sulu_admin.logout
-   +             two_factor:
-   +                 prepare_on_login: true
-   +                 prepare_on_access_denied: true
-   +                 check_path: 2fa_login_check_admin
-   +                 authentication_required_handler: sulu_security.two_factor_authentication_required_handler
-   +                 success_handler: sulu_security.two_factor_authentication_success_handler
-   +                 failure_handler: sulu_security.two_factor_authentication_failure_handler
+   +            two_factor:
```

Comment:
The additional space here is expected because it shows a `git diff` if you copy the lines into your file you and remove the `+` it will have the correct indention.

Reply:
Hm, the sphinx diff rendering seems to be a bit confusing then, at least for me :)

Diff on your PR in Github (https://github.com/sulu/skeleton/commit/1572f8aefccb6686cb6a3dbcf5dd466ff1788def?diff=unified):
![image](https://user-images.githubusercontent.com/5249624/179549526-cfe9bdf7-8701-45b8-9661-5179e74e7f1a.png)

Diff in the Sulu Docs (https://docs.sulu.io/en/latest/bundles/security/index.html?highlight=2fa#two-factor-authentication):
![image](https://user-images.githubusercontent.com/5249624/179549642-be2b88cf-7bc4-467b-bc00-f0f1f40f9833.png)

Maybe there is some config that can be changed to show it aligned?

Reply:
Sadly the copy & paste view in github is for most diff views completely broken and copy false parts. Atleast copy & paste in sphiny renderer works like expected without any problems here. It just a little bit confusing with indention when looking at the code but I would currently keep it this way to make it easier to copy it instead of go on how it looks.